App/components/Settings/SettingsDropdown.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { Modal, Share, SafeAreaView } from 'react-native';
+import * as MailComposer from 'expo-mail-composer';
+import styled from '@emotion/native';
+
+import { notify } from 'lib/Tracking';
+import { useI18n } from 'lib/hooks';
+import { selectors } from 'store';
+
+export default function SettingsDropdown({ navigation }) {
+  const { i18n } = useI18n();
+  const dispatch = useDispatch();
+  const showDropdown = useSelector(selectors.showDropdown);
+
+  const setShowDropdown = () => {
+    if (showDropdown) dispatch({ type: 'HIDE_DROPDOWN' });
+    else dispatch({ type: 'SHOW_DROPDOWN' });
+  };
+
+  async function onShare () {
+    try {
+      const result = await Share.share({
+        url: 'https://github.com/House-of-Faith/RN-Chinese-Bible',
+      });
+      if (result.action === Share.sharedAction) {
+        if (result.activityType) {
+          // shared with activity type of result.activityType
+        } else {
+          // shared
+        }
+      } else if (result.action === Share.dismissedAction) {
+        // dismissed
+      }
+    } catch (error) {
+      notify(error);
+    }
+  }
+
+  async function onFeedback () {
+    try {
+      const result = await MailComposer.isAvailableAsync();
+      console.log(result);
```

Comment:
Looks like a log statement was left in.

Reply:
That feature is incomplete. The log has been there a while